doc/concepts/labels.rst

```diff
@@ -870,4 +870,3 @@ API
 ---
 
 * :ref:`Python <pylabels>`
-* :ref:`C++ <cpplabels>`
```

Comment:
I really want to keep these in, because users should know that stuff can be done in both lagnuages. Them being missing is something I don't object to showing, and otherwise we should track separately the missing bits which in the past proved to go out of date too quickly.